cli/pcluster/cfnconfig.py

```diff
@@ -516,6 +516,7 @@ def __init_cluster_parameters(self):
             custom_chef_cookbook=("CustomChefCookbook", None),
             additional_cfn_template=("AdditionalCfnTemplate", None),
             custom_awsbatch_template_url=("CustomAWSBatchTemplateURL", None),
+            ec2_iam_policy=("EC2IAMPolicy", "EC2IAMPolicy"),
```

Comment:
shall we pick a name that suggests the fact that this is a policy added to the existing set of default policies, such as additional_ec2_iam_policy? Also what about having a list of policies so that users can combine multiple policies or AWS managed policies?

Reply:
Updated to `additional_iam_policy`